core/templates/pages/admin-page/misc-tab/admin-misc-tab.component.html

```diff
@@ -14,6 +14,11 @@ <h3>Search Index</h3>
   <button (click)="clearSearchIndex()">Clear Search Index</button>
 </mat-card>
 
+<mat-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h3>Publishing the chapters of Length and Measurement topic</h3>
```

Comment:
Publish the chapters of the Length Measurements topic.

Reply:
Done!